frontend/test/metabase/scenarios/admin/databases/list.cy.spec.js

```diff
@@ -8,7 +8,18 @@ describe("scenarios > admin > databases > list", () => {
     cy.server();
   });
 
-  it("should let you view a database", () => {
+  it("should let you see databases in list view", () => {
+    cy.visit("/admin/databases");
+    cy.findByText("Sample Dataset");
+    cy.findByText("H2");
+  });
+
+  it("should not let you see saved questions in the database list", () => {
+    cy.visit("/admin/databases");
+    cy.get("td").should("have.length", 3);
```

Comment:
What's in the `td`s? Could we check for text/icons instead?

Reply:
`td` are the rows within the list (3 tds comprise each item)-- There aren't any icons and I assumed testing `td` would be better because no matter what the saved are that appear, they will cause the td number to increase. What do you think?

Reply:
Sorry I totally glanced over the test name 🤦 

I think we should probably assert specifically that "Saved Questions" isn't present. Counting `td`s would break if we added a column or refactored the display to not use `<table>` for some reason. 

Also, IMO it could be combined with the preceding test. It doesn't really matter, but it'd save us a page load.

Reply:
🤔 actually maybe we could just not call `cy.visit` and rely on the page being loaded? I'm not sure what's the cypressy way to do things, but probably not worth worrying about a second or two.

Reply:
> 🤔 actually maybe we could just not call `cy.visit` and rely on the page being loaded
Don't we want to load the page in order to test the front end? How would we do that without loading the page?